docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+services:
+  app:
+    restart: always
+    image: docker-rwd:tag
```

Comment:
With the `ENTRYPOINT` added back in, `docker run --rm -ti docker-rwd` works with or without the tag. Is it still preferable to remove it?